src/mscorlib/shared/System/Globalization/TimeSpanParse.cs

```diff
@@ -487,14 +488,41 @@ internal bool SetFailure(ParseFailureKind kind, string resourceKey, object messa
                     case ParseFailureKind.FormatWithParameter:
                         throw new FormatException(SR.Format(message, messageArgument));
 
-                    case ParseFailureKind.Overflow:
-                        throw new OverflowException(message);
-
                     default:
                         Debug.Assert(kind == ParseFailureKind.Format, $"Unexpected failure {kind}");
                         throw new FormatException(message);
                 }
             }
+
+            internal bool SetOverflowFailure()
+            {
+                if (!_throwOnFailure)
+                {
+                    return false;
+                }
+
+                throw new OverflowException(SR.Format(SR.GetResourceString(nameof(SR.Overflow_TimeSpanElementTooLarge)), new string(_originalTimeSpanString)));
```

Comment:
is'nt this
```c#
throw new OverflowException(SR.Format(SR.Overflow_TimeSpanElementTooLarge, new string(_originalTimeSpanString)));
```
and below? should not need to do `nameof(SR.` anywhere.

Reply:
Correct.